Core/clim-core/panes/stream-panes.lisp

```diff
@@ -79,6 +77,15 @@
    "This class implements a pane that supports the CLIM graphics,
     extended input and output, and output recording protocols."))
 
+(defmethod initialize-instance :after ((instance clim-stream-pane) &rest initargs)
+  (declare (ignore initargs))
+  (with-slots (scroll-bars label) instance
+    (when (or (not (eql :obsolete scroll-bars))
+              (not (eql :obsolete label)))
+      (warn "LABEL and SCOLL-BARS slots in CLIM-STREAM-PANE are
+      obsolete, don't use them but use the keywords :LABEL and
+      :SCROLL-BARS in function MAKE-CLIM-STREAM-PANE"))))
```

Comment:
Please split this into two warnings so it is clear which one was supplied.

Reply:
done